### ISSUE TITLE
UI/UX enhancement to project dialog box - editable path widget

### ISSUE THREAD

**Issue by gb119** [spyder-ide/spyder#16817]
A suggestion for a minor enhancement to the New/Open project dialog box - the current dialog box's path widget seems (to me anyway) to invite the user to type a path *or* to click the folder button to browse for a path - but actually typing a path isn't possible. It would be better if that widget allowed both modes of data entry - and even better did tab completion when typing.

As ever, thanks for all the work from the team for making a nice IDE.

**Comment by steff456** [comment]
Hi @gb119,

I have to agree with you! I also wasn't aware that users are unable to type in the dialog box. We can definitely work in improving this interaction, what do you think @spyder-ide/core-developers ?

**Comment by mrclary** [comment]
I think this is a good idea.

**Comment by dalthviz** [comment]
+1 from my side too

**Comment by CAM-Gerlach** [comment]
Makes sense; if it looks like a combobox, it should work like a combobox. Thanks for the suggestion, @gb119 :+1:

**Comment by ccordoba12** [comment]
This was addressed in the revised `New Project` dialog added in 6.1.0:

<img width="735" height="482" alt="Image" src="https://github.com/user-attachments/assets/205792e7-03d3-4fd8-a6ee-40c954976a3f" />

**Comment by gb119** [comment]
I noticed it - thanks for implementing!

**Comment by CAM-Gerlach** [comment]
Changing close state to completed, since this was implemented rather than rejected
